### ISSUE TITLE
 KW-0548  and other annotations for Q10315

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#2236]
Q10315 is no longer considered to be fission yeast dpb3.

dpb3 is
https://www.pombase.org/gene/SPCC16C4.22

I think the explanation is in here IIRC
https://www.pombase.org/reference/PMID:29109278

this is actually pombe bur6 ortholog (a transcription factor)

annotations from
https://www.ncbi.nlm.nih.gov/pubmed/15388803
should be removed

**Comment by ValWood** [comment]
Hi @marcfeuermann  you might not be the correct person for this one so could you reassign?

Thanks

Val

**Comment by marcfeuermann** [comment]
Hi Val,
As I'm still 50% UniProtKB curator for fungal proteins, so you assigned this ticket correctly to me :-) 
This is an unusual situation of identity theft ! I've made the changes in UniProtKB and SPAC17G8.03c (Q10315) is now the transcription regulator complex subunit bur6, SPCC16C4.22 (C6Y4D0) having now the identity of dpb3. As I've changed the KWs, mapping to GO should also change.
Regards,
Marc.
